### DISCUSSION TITLE
Command path

### DISCUSSION THREAD

**Discussion by mdicss** [Linuxfabrik/monitoring-plugins#683] [Q&A] [answered]
Hi
In the command definitions (director baskets) you use the path '/usr/lib64/nagios/plugins' Would it not be better to always use 'PluginDir' to be more portable?
Maybe it is a json problem (quotation marks)?
Regards
Matthias

**Comment by markuslf** [comment, marked as answer]
Of course, yes, but have a look at the note here: https://github.com/Linuxfabrik/monitoring-plugins/blob/main/INSTALL.rst#sudoers

**Comment by mdicss** [comment]
Ok, now I've seen https://github.com/Icinga/icingaweb2-module-director/issues/2123.
So I just changed the path in the basket with an editor.
